lib/vsc/utils/fancylogger.py

```diff
@@ -431,7 +438,7 @@ def getRootLoggerName():
         return "not available in optimized mode"
 
 
-def logToScreen(enable=True, handler=None, name=None, stdout=False):
+def logToScreen(enable=True, handler=None, name=None, stdout=False, color=False):
```

Comment:
Just tell me what you would like to use: 'auto'/'yes'/'no' (all strings)? True/False/None? 0/1/2? named constants?